README.md

```diff
@@ -1,7 +1,7 @@
 [![build](https://github.com/dianna-ai/dianna/actions/workflows/build.yml/badge.svg)](https://github.com/dianna-ai/dianna/actions/workflows/build.yml)
 [![workflow pypi badge](https://img.shields.io/pypi/v/dianna.svg?colorB=blue)](https://pypi.python.org/project/dianna/)
 [![workflow scc badge](https://sonarcloud.io/api/project_badges/measure?project=dianna-ai_dianna&metric=coverage)](https://sonarcloud.io/dashboard?id=dianna-ai_dianna)
-[more badges](badges.md)
+[![more badges badge](https://img.shields.io/badge/more-badges-yellow)](badges.md)
```

Comment:
yellow is implying that something could have been green if the dev's of this repo had tried harder ;-)
```suggestion
[![more badges badge](https://img.shields.io/badge/more-badges-lightgrey)](badges.md)
```

Reply:
Yellow is gold! But fine ;)